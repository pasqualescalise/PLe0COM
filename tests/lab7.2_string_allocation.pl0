BEGIN
	new(x, char, 20);
	x := "Hello World!";
	print x;
	x := "Hello";
	print x;
	x := "World";
	print x;
	x := "AAAAAAAAAAAAAAAAAAA";
	print x
END.
