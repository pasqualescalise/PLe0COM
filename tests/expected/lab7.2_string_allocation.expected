Hello World!
Hello
World
AAAAAAAAAAAAAAAAAAA
